src/main/webapp/scripts/map.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const MAP_THEME = [
+  {
+    "featureType": "poi.business",
+    "stylers": [
+      {
+        "visibility": "off"
+      }
+    ]
+  },
+  {
+    "featureType": "poi.park",
+    "elementType": "labels.text",
+    "stylers": [
+      {
+        "visibility": "off"
+      }
+    ]
+  }
+];
+
+/**
+ * Map declaration and initial setup
+ * At the beginning, retrieves the API_KEY from an external file
+ */
+$(function () {
+  jQuery.get('../../../../API_KEY.txt', function (textString) {
+    let API_KEY = textString;
+
+    // Create the script tag, set the appropriate attributes.
+    var script = document.createElement('script');
```

Comment:
According to the JavaScript guidelines `var` should never be used. Also I think this name could seem a little more descriptive, what do you think?
```suggestion
    var scriptTag = document.createElement('script');
```

Reply:
Sure, I'll change that